.github/workflows/shell-script.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: execute shell script
-      - uses: actions/github-script@v4
+        uses: actions/github-script@v4
```

Comment:
Are two uses in the same step allowed?
It  looks like we are not using github-script, so we could remove the uses statement fully.

Reply:
So `run: git log --format='%an'` instead of the uses step?
we can use two uses but only the first use can use a `-`

Reply:
Error: Unhandled error: SyntaxError: Unexpected identifier
It looks like github-script is javascript, so the below shell script error-ed out.

Reply:
Is this changes okay?
`steps:`
      `- uses: actions/checkout@v2`
      `- name: print author name`
      `- run: git log --format='%an'`